cvp_checks/utils/__init__.py

```diff
@@ -17,6 +17,18 @@ def cmd(self, tgt, fun, param=None, expr_form=None, tgt_type=None):
         if param:
             accept_key_payload['arg'] = param
 
+        try:
+            if "http_proxy" in os.environ:
+                raise Exception("PROXY_SET")
+            if "https_proxy" in os.environ:
+                raise Exception("PROXY_SET")
```

Comment:
What if no_proxy is defined?

Reply:
check my latest commit plz.
the goal is to warn user about proxy variables set and go ahead with test run. So no interupton, it just alert in console about http_proxy, https_proxy and no_proxy variables and continue.
Why so? not always proxy configured by default so user needs to configure it manually or start cvp_checks alongside/separately from cvp pipeline.
It can be handled with no exception raise, but what is the proper way then?